### ISSUE TITLE
DOM manipulation

### ISSUE THREAD

**Issue by etxemag75** [novaframework/nova#289]
I like your approach with Nova, but I don't see any support to dynamically modify html objects in the DOM after the page has been served, but I could have missed it when went through the examples and Nova framework code.

Other frameworks like N2O opens up a WebSocket connection between the Server and the Web browser, which is then used to send javascript commands to update DOM,  dynamically adding or removing html objects. Also Nitrogen framework uses a similar approach with Ajax.

Are you planning to add something similar?

**Comment by Taure** [comment]
We have discussed it before, we will probably not add into Nova. But maybe have something built using Nova. Try to keep Nova still small and with core functionality.

We did some minor projects before to try to build something that could work with Nitrogen.

I am open to discussion around this topic and would love to see DOM updates in applications using Nova. I think it would be really good and fun to have.
